receiver/hostmetricsreceiver/internal/scraper/processscraper/process_scraper.go

```diff
@@ -250,7 +250,7 @@ func (s *processScraper) getProcessMetadata(ctx context.Context) ([]*processMeta
 		}
 
 		command, err := getProcessCommand(ctx, handle)
-		if err != nil {
+		if err != nil && !s.config.MuteProcessAllErrors {
```

Comment:
Why do we add muting here  but the related change on line 277 still unmuted? Maybe remove this from this PR to fix consistently separately?

Reply:
d43aa79684